spec/features/visitors/sign_up_spec.rb

```diff
@@ -82,24 +89,8 @@
       expect(current_path).to eq users_otp_path
     end
 
-    it 'disables OTP lockout during account creation' do
```

Comment:
Actually I'm not sure about testing for a negative here.   Why would we tests that phone confirmation is not locked out after N attempts?  What would we set N too?  There are an infinite number of things that we could test that `#confirm` does *not* do. I don't see why we should single out a random value of N and check that N attempts *doesn't* lock the user out.